components/screens/TokenOperation.tsx

```diff
@@ -9,6 +9,7 @@ import { setPopupPath } from "@/lib/utils.ts";
 import { useTokenInfo } from "@/hooks/useTokenInfo.ts";
 import { applyDecimal } from "@/lib/krc20.ts";
 import { useLocation } from "react-router";
+import { State } from "@/components/screens/Send.tsx";
```

Comment:
I think `State` is too generic to be exported as is. Maybe rename to `SendState`.